src/instructlab/utils.py

```diff
@@ -541,3 +541,20 @@ def split_hostport(hostport: str) -> tuple[str, int]:
     if not hostname or not port:
         raise ValueError(f"Invalid host-port string: '{hostport}'")
     return hostname, port
+
+
+def is_oci_repo(repo_url: str) -> bool:
+    """
+    Checks if a provided repository follows the OCI registry URL syntax
+    """
+
+    # TO DO: flesh this out and make it a more robust check
+    oci_url_prefix = "docker://"
```

Comment:
so there are a few different type of prefixes here. I think `docker://` is ok for a starter. I think `oci://` is also one though? unsure honestly

Reply:
ack, I can add `oci://` as well

Reply:
oci normally refers to oci directories vs registries, but I have some references to a different interpretation although from my understanding not widespread. Relatedly see https://github.com/containers/skopeo/issues/633

if we do support that convention we would need to remap it to a docker url scheme before passing off to skopeo

Reply:
thanks, jason
I'll probably leave it as is for now then